docs/markdown/06-iterator_generator/05-AsyncIterator.md

```diff
@@ -0,0 +1,50 @@
+<!-- .slide:-->
+
+# Itérables Asynchrones
+
+Un itérable asynchrone est un iterable avec un itérateur dont la méthode next retourne une Promesse de {done, value} au lieu d'un objet {done, value}.
+<!-- .element: class="fragment" -->
+Un objet est considéré comme **itérable asynchrone** s'il définit le comportement qu'il aura lors de l'itération asynchrone
+<!-- .element: class="fragment" -->
+Pour qu'un objet soit **itérable asynchrone**, un objet doit implémenter la méthode **@@asyncIterator** (qui est un **Symbol**)
+<!-- .element: class="fragment" -->
+
+##==##
+
+# Itérables Asynchrones : Comment les parcourir
```

Comment:
je rajouterais un code de classe itérable asynchrone (peut être reprendre le code de l'itérateur vu dans le chapitre d'avant ?)

Reply:
car au final dans le code que tu mets ensuite, comme tu utilise Array, on ne voit pas l'implémentation du @@asyncInterator